src/main/java/de/kittybot/kittybot/WebService.java

```diff
@@ -108,17 +107,19 @@ private static void login(Context ctx){
 
 	private static void logout(Context ctx){
 		var auth = ctx.header("Authorization");
-		if(auth != null){
```

Comment:
i'd keep this check as it was before, it was shorter

Reply:
Yeah true